physionet-django/events/models.py

```diff
@@ -138,3 +138,49 @@ def withdraw(self, comment_to_applicant=None):
             comment_to_applicant (str): The comment to add to the participant about the status.
         """
         self._apply_decision(self.EventApplicationStatus.WITHDRAWN, comment_to_applicant)
+
+
+class EventDataset(models.Model):
+    """
+    Captures information about datasets for events.
+    """
+    class EventDatasetStatus(models.TextChoices):
+        Active = 'AC', _('Active')
+        Inactive = 'IN', _('Inactive')
+
+    class EventDatasetAccessType(models.TextChoices):
+        GOOGLE_BIG_QUERY = 'GBQ', _('Google BigQuery')
+        RESEARCH_ENVIRONMENT = 'RE', _('Research Environment')
+
+    event = models.ForeignKey(Event, on_delete=models.CASCADE, related_name='datasets')
+    dataset = models.ForeignKey("project.PublishedProject", on_delete=models.CASCADE)
+    access_type = models.CharField(default=EventDatasetAccessType.GOOGLE_BIG_QUERY, max_length=3,
```

Comment:
3 characters seem short. I'd make this 10 just to save effort in the future. For example we might want `AWS-S3`, etc.